spec/models/booking_spec.rb

```diff
@@ -30,6 +30,58 @@
 
       expect(Booking.last_week.count).to eq(2)
     end
+
+    it 'should return bookings with bondable charges' do
```

Comment:
Or we could nest another layer of `describe` blocks, like you do in `person_spec`. Either works.